definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js

```diff
@@ -0,0 +1,31 @@
+import type React from 'react';
+
+type ReactModal$DefaultProps = {
+  isOpen: bool,
+  ariaHideApp: bool,
+  closeTimeoutMS: number,
+  shouldCloseOnOverlayClick: bool,
+}
+
+type ReactModal$Props = {
+  isOpen: bool,
+  style?: {
+    content?: Object,
+    overlay?: Object,
+  },
+  appElement?: HTMLElement,
+  onRequestClose?: Function,
```

Comment:
Looks like there's an [onAfterOpen](https://github.com/reactjs/react-modal/blob/master/lib/components/Modal.js#L34) prop in there, too